scripts/gulpfiles/build_tasks.js

```diff
@@ -116,7 +116,8 @@ function compile(compilerOptions, opt_verbose, opt_warnings_as_error,
   const options = {};
   options.compilation_level = 'SIMPLE_OPTIMIZATIONS';
   options.warning_level = opt_verbose ? 'VERBOSE' : 'DEFAULT';
-  options.language_in = 'ECMASCRIPT5_STRICT';
+  // Use stable, regardless of what was passed in.
+  options.language_in = 'STABLE';
```

Comment:
Any reason to explicitly set this if this is the default?

Reply:
Paranoia that it could be called in other parts of our gulpfiles that I didn't catch.

But I can check that explicitly and then remove this.